spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -11,6 +11,192 @@ We then build an introductory Spring Cloud Data Flow application, discussing som
 
 
 
+[[getting-started-deploying-spring-cloud-dataflow-quick-start]]
+== Quick Start using Docker Compose
+
+As of Spring Cloud Data Flow 1.4, a Docker Compose file is provided to quickly bring up Spring Cloud Data Flow and its dependencies without having to obtain them manually. Docker Compose is required and it's recommended to use the link:https://docs.docker.com/compose/install/[latest version].
+
+. Download the Spring Cloud Data Flow Local Server Docker Compose file:
++
+[source,bash,subs=attributes]
+----
+wget https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/master/spring-cloud-dataflow-server-local/docker-compose.yml
+----
++
+. Start Docker Compose
++
+In the directory where you downloaded `docker-compose.yml`, start the system, as follows:
++
+[source,bash,subs=attributes]
+----
+$ docker-compose up
+----
++
+. Launch the Spring Cloud Data Flow Dashboard
++
+Spring Cloud Data Flow will be ready for use once the `docker-compose` command stops emitting log messages.
+At this time, in your browser navigate to the link:http://localhost:9393/dashboard[Spring Cloud Data Flow Dashboard].
+By default the latest GA Stream and Task applications will be imported automatically.
++
+. Create a Stream
++
+To create a stream, first navigate to the "Streams" menu link then click the "Create Stream" link.
+Enter `time | log` into the "Create Stream" textarea and click the "CREATE STREAM" button.
+Enter "ticktock" for the stream name and click the "Deploy Stream(s) checkbox, then click "OK".
+The stream will be in "deploying" status and move to "deployed" when finished.
+You may need to refresh your browser to see the updated status.
++
+. View Stream Logs
++
+To view the stream logs, navigate to the "Runtime" menu link and click the "ticktock.log" link.
+Copy the path in the "stdout" text box on the dashboard and in another console type:
++
+[source,bash,subs=attributes]
+----
+$ docker exec -it dataflow-server tail -f /path/from/stdout/textbox/in/dashboard
+----
++
+You should now see the output of the log sink, printing a timestamp once per second.
+Press CTRL+c to end the `tail`.
++
+. Delete a Stream
++
+To delete the stream, first navigate to the "Streams" menu link in the dashboard then click the checkbox on the "ticktock" row.
+Click the "DESTROY ALL 1 SELECTED STREAMS" button and then "YES" to destroy the stream.
++
+. Destroy the Quick Start environment
++
+To destroy the Quick Start environment, in another console, type as follows:
++
+[source,bash,subs=attributes]
+----
+$ docker-compose down
+----
++
+
+
+[[getting-started-customizing-spring-cloud-dataflow-quick-start]]
+== Quick Start using Docker Compose Customization
```

Comment:
This doesn't have to be a top-level section. We can make it a sub-section under "Getting Started with Docker Compose".

Reply:
I like overall structure of this section. Easy to follow.